src/Recurrence.ts

```diff
@@ -219,4 +220,23 @@ export class Recurrence {
 
         return null;
     }
+
+    public identicalTo(other: Recurrence) {
+        if (this.baseOnToday !== other.baseOnToday) {
+            return false;
+        }
+
+        // Compare Date fields
+        if (Sort.compareByDate(this.startDate, other.startDate) !== 0) {
```

Comment:
`moment` has a function [`isSame()`](https://momentjs.com/docs/#/query/is-same/).